plugins/module_utils/identity/bitwarden.py

```diff
@@ -0,0 +1,321 @@
+# -*- coding: utf-8 -*-
+
+# (c) 2022, Jonathan Lung (@lungj) <lungj@heresjono.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import json
+
+from ansible.module_utils.common.text.converters import to_text
+
+
+class BitwardenException(Exception):
+    pass
+
+
+class Client(object):
+    '''A wrapper for for bw command.'''
+
+    # Path to the `bw` executable.
+    cli_path = 'bw'
+    # AnsibleModule using the client.
+    module = None
+
+    @staticmethod
+    def run(args, stdin=None, expected_rc=0):
+        '''Run the bw client.
+
+        Args:
+            args(list): Arguments to pass to the bw CLI client.
+            stdin(str): Data to write to the client's stdin.
+            expected_rc(int): Expected return code of bw CLI client.
+
+        Return:
+            (str, str): The contents of standard out and standard error.
+        '''
+        rc, out, err = Client.module.run_command([Client.cli_path] + args, data=stdin, binary_data=True)
+
+        if rc != expected_rc:
+            raise BitwardenException(err)
+        return to_text(out, errors='surrogate_or_strict'), to_text(err, errors='surrogate_or_strict')
+
+    @staticmethod
+    def run_json(args, stdin=None, expected_rc=0):
+        '''Run the bw client.
+
+        Args:
+            args(list): Arguments to pass to the bw CLI client.
+            stdin(str): Data to write to the client's stdin.
+            expected_rc(int): Expected return code of bw CLI client.
+
+        Return:
+            dict: Decoded dictionary object returned by bw CLI client.
+        '''
+        stdout, _stderr = Client.run(args, stdin, expected_rc)
+        return json.loads(stdout)
+
+
+class _Element(object):
+    '''Common elements for items in a Bitwarden vault.'''
+
+    id = property(lambda self: self._id or self.dict['id'])
+    name = property(lambda self: self._name or self.dict['name'])
+
+    _dict = None
+
+    @property
+    def exists(self):
+        # Determine existence by attempting to access the item.
+        try:
+            obj_dict = self.dict
+        except KeyError:
+            return False
+        # Wildcard objects can return a dict, but their name is None.
+        return obj_dict['name'] is not None
+
+    @property
+    def dict(self):
```

Comment:
Naming the property with symbol used by Python is not a good practice. I recommend renaming it to something like `as_dict()` or `to_dict()`, or change it entirely to something related to attributes, as mentioned elsewhere.